src/L2GovernorMetadata.sol

```diff
@@ -17,6 +18,26 @@ abstract contract L2GovernorMetadata {
   /// @notice The id of the proposal mapped to the proposal metadata.
   mapping(uint256 proposalId => Proposal) _proposals;
 
+  /// @notice The assumed block time of the base network
+  uint256 private L1_BLOCK_TIME = 12;
+  /// @notice The assumed block time of the aux network
+  /// @dev These are hardcoded now for Ethereum mainnet & Optimism, as these are currently
+  /// the target networks for the MVP launch. In the future, this should be generalized to work
+  /// for different network combinations. Even better, once we have better support for cross chain
+  /// voting in clients and frontend tools, this hack should removed completely.
+  uint256 private L2_BLOCK_TIME = 2;
+
+  /// @notice The contract that handles fetch the L1 block on the L2.
```

Comment:
fetching instead of fetch

Reply:
Done in 165e503186e3748ec195858fcb3972d1021c0610